src/main/java/seedu/address/newstorage/JsonSerializableRecipeBook.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.newstorage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.ReadOnlyRecipeBook;
+import seedu.address.model.RecipeBook;
+import seedu.address.model.recipe.Recipe;
+
+/**
+ * An Immutable AddressBook that is serializable to JSON format.
```

Comment:
minor: update the JavaDocs in this file too
```suggestion
 * An Immutable RecipeBook that is serializable to JSON format.
```